parser.py

```diff
@@ -623,6 +632,13 @@ def main():
 
     users = {}
 
+    # Use our state store to prevent duplicate downloads
+    try:
+        with open(state_path, 'r') as state_file:
+            state = json.load(state_file)
```

Comment:
`state` is a very general word. Would `cache` be better?

Reply:
When it started out, it was only about keeping download state as the downloaded images were already on disk.
So we were not really caching any data.

But now that we're actually handling user data, sure. We can relabel to cache.